.pre-commit-config.yaml

```
@@ -12,7 +12,7 @@ repos:
     hooks:
     - id: system
       name: flake8
```

Better yet, maybe call it `lint`?

```suggestion
      name: ruff
```

---

Again, automerge 🙈

---

Same